sos/report/plugins/grub2.py

```diff
@@ -39,7 +39,7 @@ def setup(self):
             "/etc/grub2-efi.cfg"
         ])
 
-        self.add_cmd_output("ls -lanR /boot", tags="ls_boot")
+        self.add_dir_listing('/boot', tags=['ls_boot'])
```

Comment:
just thinking, we have the listing of `/boot` here and the boot plugin, is it worth removing it from one of the plugins?

Reply:
Yeah, I thought about dropping it but instead had this commit focus on changing over to the new method, rather than consolidating collections. It's a simple enough change though, so I think we'd be fine dropping it from `grub2`.